frontend/src/static/js/v_authorship.js

```diff
@@ -147,6 +161,34 @@ window.vAuthorship = {
           }), {});
     },
 
+    selectAll() {
+      if (!this.allSelected) {
+        this.filesShown = this.fileTypes;
```

Comment:
Perhaps rename to `fileTypesSelected` (abit confused between `filesShown` and `filesSelected`)?